VENTokenField/VENTokenField.m

```diff
@@ -311,7 +311,12 @@ - (void)layoutToLabelInView:(UIView *)view origin:(CGPoint)origin currentX:(CGFl
     self.toLabel.frame = newFrame;
     
     [view addSubview:self.toLabel];
-    *currentX += self.toLabel.hidden ? CGRectGetMinX(self.toLabel.frame) : CGRectGetMaxX(self.toLabel.frame) + self.toLabelPadding;
+
+    if ([self.dataSource respondsToSelector:@selector(marginBetweenToFieldAndTokens)]) {
+        *currentX += [self.dataSource marginBetweenToFieldAndTokens];
+    } else {
+        *currentX += self.toLabel.hidden ? CGRectGetMinX(self.toLabel.frame) : CGRectGetMaxX(self.toLabel.frame) + self.toLabelPadding;
+    }
```

Comment:
I think this should continue to use the existing format of a property on VENTokenField that can be overriden by the delegate or can be set like toLabel. I would think this code would stay the same but we'd add the margin in.